docs/getting_started.md

```diff
@@ -2,12 +2,37 @@
 
 ## Installation
 
-Install Cubed with pip:
+### Conda
+
+You can install cubed with a minimal set of dependencies using conda:
+
+```shell
+conda install -c conda-forge cubed
+```
+
+### Pip
+
+You can also install cubed with pip:
 
 ```shell
 python -m pip install cubed
 ```
 
+Cubed has many optional dependencies, which can be installed in sets for different functionality (especially for running on different executors):
+
+    $ python -m pip install "cubed[diagnostics]"  # Install optional dependencies for cubed diagnostics
+    $ python -m pip install "cubed[beam]"         # Install optional dependencies for the beam executor
+    $ python -m pip install "cubed[lithops]"      # Install optional dependencies for the lithops executor
+    $ python -m pip install "cubed[modal]"        # Install optional dependencies for the modal executor
+    $ python -m pip install "cubed[complete]"     # Install all of the above
```

Comment:
I just remembered that Beam, Lithops and Modal don't work together due to package conflicts, so we shouldn't have a `complete` option.